packages/commonwealth/client/scripts/hooks/useInitChainIfNeeded.ts

```diff
@@ -10,10 +10,11 @@ import { initChain } from 'helpers/chain';
  * Helper function initChain requires app.chain to be selected, otherwise this will do nothing.
  * So this will run after the chain is selected, but not if the chain is loaded already.
  */
-export const useInitChainIfNeeded = (app: IApp) => {
+export const useInitChainIfNeeded = (app: IApp, ignore = false) => {
```

Comment:
Needed because if we execute this function, we can't load proposals in thread (Haven't looked into it, I just know it happens).